src/commands/pull.ts

```diff
@@ -71,6 +73,12 @@ export default new Command()
       .default('.')
       .argParser((v) => v || '')
   )
+  .addOption(
+    new Option(
+      '-n, --namespace <namespace...>',
+      'Namespace, pull keys from specified namespaces. Defaults all namespaces'
+    ).argParser((v, a: string[]) => [v, ...(a || [])])
```

Comment:
This parser isn't needed here. It was used in the states argument to turn the argument always uppercase as expected by the API; but for example the languages does not use this.